src/config.ts

```diff
@@ -157,20 +133,15 @@ export function getSmartAgentApiKey(): string {
   return process.env.SMARTAGENT_API_KEY || "";
 }
 export function getSmartAgentApiUrl(): string {
-  return process.env.SMARTAGENT_API_URL || "";
+  return (
+    process.env.SMARTAGENT_API_URL || "https://de2ghtit0q15h.cloudfront.net"
```

Comment:
Don't suppose you could add a comment here explaining what     `https://de2ghtit0q15h.cloudfront.net` is? I assume it's a URL used for localhost work?
```suggestion
    process.env.SMARTAGENT_API_URL || "https://de2ghtit0q15h.cloudfront.net" # default to endpoint for local work
```